tests/testthat/test-txt_wrap.R

```diff
@@ -54,38 +54,72 @@ test_that("toString() throws a warning when newline is in string", {
 
 test_that("works with words that are too big (no warning)", {
     bmf <- basic_matrix_form(mtcars[1:2, 1:2])
-    bmf$col_gap <- 3
-    bmf$table_inset <- 1
-    bmf$main_title <- strrep("a", 30)
```

Comment:
Please use the setters (e.g., `main_title(bmf) <- strrep("a", 30)`, etc) instead of direct assignment so that these tests are robust to us changing the MatrixPrintForm class in the future (which may or may not happen, but its a good idea not to penetrate abstraction when you don't need to for that reason).

Other than that this looks good